gnutls-v3.5.0/Readme.md

```diff
@@ -0,0 +1,44 @@
+Finds a SEGV error during parsing of a corrupted certificate, in
```

Comment:
Same here: please indicate how much time is needed to find this bug. 
If it's several seconds, it's probably not that interesting.